ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vrfs.j2

```diff
@@ -37,6 +37,7 @@
 {%                 if configure_mlag_ibgp_peering %}
         {{ switch.mlag_peer_l3_ip | arista.avd.default(switch.mlag_peer_ip) }}:
           peer_group: {{ bgp_peer_groups.MLAG_IPv4_UNDERLAY_PEER.name | arista.avd.default("MLAG-IPv4-UNDERLAY-PEER") }}
+          description: {{ switch.mlag_peer | arista.avd.default(switch.mlag_peer_l3_ip, switch.mlag_peer_ip) }}
```

Comment:
This looks good. But how about adding a prefix to this description **MLAG_PEER_** or something similar? Let me know your opinion.

Reply:
I thought about it, but in the MLAG peer for the default VRF it only has the peer name, so I decided to leave it the same way for consistency. 
We could change it everywhere and also add the VRF name, for example. I don't think it's very useful because in majority of leaves it will be the only BGP peer on the VRF.

Reply:
That's fine. We need to maintain consistency.